recipes/combo/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "combo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3c5da2b8af17e03146ce382d10150373a260ce64565963f7c0b0cecfc7d093be
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - matplotlib-base
+    - numba >=0.35
+    - numpy >=1.13
+    - python
+    - scikit-learn >=0.20
+    - scipy
+
+test:
+  imports:
+    - combo
+    - combo.models
+    - combo.utils
+
+about:
+  home: "https://github.com/yzhao062/combo"
+  license: BSD
```

Comment:
This is a minor comment, but the license should be `BSD-2-Clause` instead.

Reply:
Done!